.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '.idea/**'
```

Comment:
This is already in `.gitignore` so it is not needed here

Reply:
It's only for CI, see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Yes. I mean that we don't need `'.idea/**'`  here (other entries are fine), because a .idea file will never be pushed (because they are in .gitignore). So only the `'.idea/**'` entry here is redundant.

Reply:
Removed.